src/components/PlaceholderWithoutTracking.jsx

```diff
@@ -136,6 +143,7 @@ class PlaceholderWithoutTracking extends React.Component {
 				className={className}
 				ref={el => (this.placeholder = el)}
 				style={styleProp}
+				{...wrapperProps}
```

Comment:
I don't think `wrapperProps` is needed here. This is the placeholder span, which is different from the wrapper span. The placeholder span is wrapped by the wrapper span and it gets replaced once the component or image are loaded. Hope it's not too confusing. :slightly_smiling_face:

Reply:
Cool - I will get this changed this weekend. Thanks for the feedback!

Reply:
@Aljullu 

Question about a bit of the code. What's the reasoning behind:

```
if ((!effect && !placeholderSrc) || visibleByDefault) {
  return lazyLoadImage;
}
```

It looks like all the other use cases get wrapped in the `<span/>` except for the ones that fall inside that conditional. I'm wondering if someone is using a placeholder and would expect it to also be wrapped in the span and have the wrapper props applied? If not, then I can just add a note in the documentation for that. If those elements should also wrap the placeholder span, then I can make that update too. 

I just wanted to get your thoughts before making any code changes. ✌️

Reply:
> It looks like all the other use cases get wrapped in the `<span/>` except for the ones that fall inside that conditional.

That's correct, the wrapper is only needed to display the placeholder image or when there is an effect. Otherwise, there is no need to render it.

> I'm wondering if someone is using a placeholder and would expect it to also be wrapped in the span and have the wrapper props applied? If not, then I can just add a note in the documentation for that.

Yeah, we should make it clear the wrapper is only rendered when `placeholderSrc` or `effect` props are defined. Checking the docs, [`wrapperClassName`](https://github.com/Aljullu/react-lazy-load-image-component#props) already has an explanation for it, but we could expand it if it's not clear.

Hope that helps @johnstonbl01.

Reply:
Thinking about it twice, to reduce confusion it might make sense to rewrite the check you posted as:
```JS
const needsWrapper = (effect || placeholderSrc) && !visibleByDefault;
if (!needsWrapper && !wrapperClassName && !wrapperProps) {
	return lazyLoadImage;
}
```
So if `wrapperClassName` or `wrapperProps` are defined, the wrapper is rendered. I think that might be less confusing, thoughts?

Feel free to handle it in this PR or I can open a new one.

Reply:
That makes sense to me. I'll take care of it in this PR - thanks for the clarification!

Reply:
Ok - should be good to go now on these changes.

- Removed wrapperProps from being passed to the placeholder
- Updated the conditional to match your above suggestion (I agree it's easier to read)
- Updated the docs for the `wrapperProps` to reflect that the wrapping span is rendered when using `placeholderSrc` and `effect`

If I missed anything, please let me know!